### ISSUE TITLE
[BUG] NFS server not responding - RWX volumes and K3S became NodeNotReady state

### ISSUE THREAD

**Issue by MohamedShaj** [longhorn/longhorn#6003]
## Describe the bug (🐛 if you encounter this issue)

Hi,

I have created a single node k3s server with storage as Longhorn, etcd as embedded ETCD and CNI as flannel in UBUNTU 18.04 (Azure cloud)

The server was up and running for two weeks successfully, suddenly the server became unresponsive, Kubectl commands not working and k3s went to NodeNotReady state

## To Reproduce

- Create UBUNTU 18.04 server in azure
- Create k3s server with embedded etcd in v1.25.4+k3s1 version
- make any dummy or heavy load app with Longhorn as dynamic storage 

##Note

We have made ETCD and Longhorn in separate EBS volumes to avoid the IO issues

## Log or Support bundle
Kernel Log:
`May 24 15:18:18 Demo-Box kernel: [1670655.742173] cni0: port 52(vethfa98589d) entered disabled state
May 24 15:18:18 Demo-Box kernel: [1670655.753874] device vethfa98589d left promiscuous mode
May 24 15:18:18 Demo-Box kernel: [1670655.753884] cni0: port 52(vethfa98589d) entered disabled state
May 24 15:18:21 Demo-Box kernel: [1670658.647720] IPv6: ADDRCONF(NETDEV_UP): veth6e07cfbf: link is not ready
May 24 15:18:21 Demo-Box kernel: [1670658.648223] cni0: port 52(veth6e07cfbf) entered blocking state
May 24 15:18:21 Demo-Box kernel: [1670658.648226] cni0: port 52(veth6e07cfbf) entered disabled state
May 24 15:18:21 Demo-Box kernel: [1670658.648402] device veth6e07cfbf entered promiscuous mode
May 24 15:18:21 Demo-Box kernel: [1670658.653907] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
May 24 15:18:21 Demo-Box kernel: [1670658.653935] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
May 24 15:18:21 Demo-Box kernel: [1670658.653997] IPv6: ADDRCONF(NETDEV_CHANGE): veth6e07cfbf: link becomes ready
May 24 15:18:21 Demo-Box kernel: [1670658.654068] cni0: port 52(veth6e07cfbf) entered blocking state
May 24 15:18:21 Demo-Box kernel: [1670658.654070] cni0: port 52(veth6e07cfbf) entered forwarding state
May 24 15:21:19 Demo-Box kernel: [1670836.700175] nfs: server 10.43.252.141 not responding, still trying
May 24 15:21:35 Demo-Box kernel: [1670852.252148] nfs: server 10.43.65.88 not responding, still trying
May 24 15:21:39 Demo-Box kernel: [1670856.860416] nfs: server 10.43.252.141 not responding, still trying
May 24 15:21:55 Demo-Box kernel: [1670872.284298] nfs: server 10.43.174.157 not responding, timed out
May 24 15:21:55 Demo-Box kernel: [1670872.408142] nfs: server 10.43.65.88 not responding, timed out
May 24 15:21:56 Demo-Box kernel: [1670872.988271] nfs: server 10.43.252.141 not responding, timed out
May 24 15:22:03 Demo-Box kernel: [1670880.476079] nfs: server 10.43.65.88 not responding, still trying
May 24 15:22:56 Demo-Box kernel: [1670933.724423] nfs: server 10.43.174.157 not responding, still trying
May 24 15:23:49 Demo-Box kernel: [1670986.717866] INFO: task influxd:27866 blocked for more than 120 seconds.`

k3s logs -> [error.log](https://github.com/longhorn/longhorn/files/11566055/error.log)

## Environment

 - Longhorn version: 1.4.0
 - Installation method (Kubectl):
 - Kubernetes distro (e.g. RKE/K3s/EKS/OpenShift) and version: k3s , v1.25.4+k3s1
   - Number of management node in the cluster: 1
   - Number of worker node in the cluster: 1 (master and worker running in same node)
 - Node config
   - OS type and version: ubuntu 18
   - CPU per node: 64
   - Memory per node: 256
   - Disk type(e.g. SSD/NVMe): SSD
   - Network bandwidth between the nodes: single node
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal): K3S (AZURE)
 - Number of Longhorn volumes in the cluster: 15

**Comment by ejweber** [comment]
Related to a discussion in the Rancher Users group on Slack and #6002. If anyone reading this isn't a member of the Rancher Users group, please feel free to join: https://slack.rancher.io/.

https://rancher-users.slack.com/archives/CC2UQM49Y/p1684999245293709
https://rancher-users.slack.com/archives/CC2UQM49Y/p1684999245293709

@MohamedShaj, can we close #6002 in favor of this one, as it seems mostly identical and doesn't have any information in it.

**Comment by MohamedShaj** [comment]
Yes 6002 is closed now, meanwhile may I know why the k3s server went to NodeNotReady state ! is anyone analysing this !

**Comment by KarloDerEchte** [comment]
I am currently experiencing the same issue with one of my Nodes in the Cluster. Any updates?

**Comment by innobead** [comment]
cc @ChanYiLin

**Comment by fatelgit** [comment]
We are seeing a similar behavior on suse micro OS and Longhorn 1.5.0 - when this is happening pods with this volume will become unresponsive file system wise but are still responding to to our livenessProbe resulting in impaired services.

**Comment by derekbit** [comment]
> We are seeing a similar behavior on suse micro OS and Longhorn 1.5.0 - when this is happening pods with this volume will become unresponsive file system wise but are still responding to to our livenessProbe resulting in impaired services.

Still an issue in v1.6.x?
